src/settings.ts

```diff
@@ -23,6 +23,7 @@ export interface Settings {
     [key: string]: string;
   };
   afterReset?: string | Array<string | CommandSpec>;
+  dumpCommand?: string;
```

Comment:
I was thinking this might be a callback like `afterReset`, perhaps: `afterAllMigrations` (and execute it when all migrations are finished, including `current.sql` where appropriate). I think it makes sense to be able to run SQL too in this hook, e.g. you might run a validation script or trigger a background job or whatever, so it makes sense to re-use the hooks code.